BLL/Services/CandidateService.cs

```diff
@@ -92,6 +92,20 @@ private IQueryable<CandidateDTO> GetCandidatesWithStatusesInformation()
                 });
         }
 
+        private IQueryable<CandidateDTOForGetAll> GetAllCandidatesWithStatusesInformation()
+        {
+            return _db.CandidatesSandboxes.FindWithSpecificationPattern(new CandidateForHRSpecification())
+                .Select(x => new CandidateDTOForGetAll
+                {
+                    ID = x.CandidateID,
+                    Status = x.Status.Name,
+                    FirstName = x.Candidate.FirstName,
+                    LastName = x.Candidate.LastName,
+                    IsInterviewedByHR = x.Candidate.Interviews.Count() > 0,
+                    IsInterviewedByTech = x.Candidate.Interviews.Count() > 1,
```

Comment:
use automapper in this case

Reply:
I will remake this method next pr. Because I need to add new fields in the near future